src/liquid/components/ld-link/ld-link.tsx

```diff
@@ -0,0 +1,62 @@
+import '../../components' // type definitions for type checks and intelliSense
+import { Component, Host, h, Prop } from '@stencil/core'
+import { getClassNames } from 'src/liquid/utils/getClassNames'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ * @part arrow - adds chevron arrow before link text
+ * @part size - sets the size of the text
+ * @part href - sets the link address
+ * @part disabled - sets the disabled state
+ * @part target - sets target of the link
+ * @part showIcon - displays chevron icon as prefix
+ */
+
+@Component({
+  tag: 'ld-link',
+  styleUrl: 'ld-link.css',
+  shadow: true,
+})
+export class LdLink {
+  /** Sets the size of the text */
+  @Prop() size?: 'sm' | 'lg'
+
+  /** Sets the link address */
+  @Prop() href?: string
+
+  /** Sets the disabled state */
+  @Prop() disabled?: boolean
+
+  /** Sets target of the link - _blank|_self|_parent|_top|framename */
+  @Prop() target?: string
```

Comment:
You can use the `cloneAttributes` utility to pass down attributes. Please have a look at how it is used in the button component. We should support all available attributes - see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes